Tests/Program.cs

```diff
@@ -1,51 +1,67 @@
 ﻿using System;
 using System.IO;
 using Circuit;
-using ComputerAlgebra;
 using System.Collections.Generic;
 using Util;
 using System.Linq;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Threading.Tasks;
 
 namespace Tests
 {
     class Program
     {
-        // TODO: Make these command line arguments.
-        static int SampleRate = 48000;
-        static int Samples = 4800;
-        static int Oversample = 8;
-        static int Iterations = 8;
-
-        static void Main(string[] args)
+        static async Task<int> Main(string[] args)
         {
-            bool test = args.Contains("--test");
-            bool benchmark = args.Contains("--benchmark");
-            bool plot = args.Contains("--plot");
+            var rootCommand = new RootCommand().WithCommand("test", "Run tests", c => c
+                                                    .WithArgument<string>("searchPath", "Files to test")
```

Comment:
Should this be named `pattern` or something instead? It's a wildcard pattern, as opposed to a path.

Reply:
Done.